tools/__tasks__/compile/hash/index.js

```diff
@@ -12,26 +12,51 @@ const mkdirpp = pify(mkdirp);
 
 const { hash, target } = require('../../config').paths;
 
-function getSrcHash(fileMapping, file) {
-    const fileHash = hasha.fromFileSync(path.resolve(target, file), { algorithm: 'md5' });
-    return Object.assign(fileMapping, {
-        [file]: path.join(path.dirname(file), fileHash, path.basename(file)),
-    });
-}
+const standardAssetsMap = () => {
+    const assets = glob.sync('**/!(*.map|*webpack*.js)', { nodir: true, cwd: target });
+    return assets.reduce((assetMap, asset) => {
+        const assetHash = hasha.fromFileSync(path.resolve(target, asset), { algorithm: 'md5' });
+        return Object.assign(assetMap, {
```

Comment:
weird that's not picked up in their eslint config (which this file should be subject to)?

if you want update the lint config to catch it (esp if it can fix them automatically) be totally happy to +1 a PR

but we should be cautious about picking up style prefs in PRs – there is just no way we're all going to agree (the repo's history is littered with 'discussions' around this 😉). v up for broad, super-strict linting, but one value of being super-strict is that whatever gets through should count as ok...

that said, it seems like it should be in the config (unless its' tricky to lint?)